new Promise(function (resolve, reject) {
	throw new Error("Whoops!");
}).catch(console.error); // Error: Whoops!

/*
â€¦Works the same way as this:
*/

new Promise(function (resolve, reject) {
	reject(new Error("Whoops!"));
}).catch(console.error); // Error: Whoops!
